mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -146,6 +176,9 @@ class RetrievalRelevance(BuiltInScorer):
 
     name: str = "retrieval_relevance"
     required_columns: set[str] = {"inputs", "trace"}
+    default_description: str = (
+        "Evaluates whether each retrieved context chunk is relevant to the input request."
+    )
```

Comment:
Following from https://github.com/mlflow/mlflow/pull/17390/files#r2299313296, can we make this an @property?